tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import GetApiServerUri from './helpers';
+import IsManager from './is_manager';
+import {
+  serverSelectedFunc
+} from 'redux/actions';
+import { RootState } from 'redux/reducers';
+// import PropTypes from "prop-types";
+
+type CreateJoinTokenProp = {
+  globalServerSelected: string,
+}
+
+type CreateJoinTokenState = {
+  name: string,
+  ttl: number,
+  token: string,
+  spiffeId: string,
+  trustDomain: string,
+  path: string,
+  message: string,
+  servers: [],
```

Comment:
`[]` is not a type? I'm curious though, how does this typecheck?

Reply:
Removed, we don't use state "servers". Type checks as any array. To specify you use the type of the elements followed by [ ]